src/main/java/Akinator.java

```diff
@@ -0,0 +1,210 @@
+            return true;
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+
+    public static void chooseYourOption() {
+        Scanner scanner = new Scanner(System.in);
+        int option = scanner.nextInt();
+
+        switch (option) {
+//Потрібно зрозуміти різницю між вибором по cssSelector-y i xpath і вибрати краще.
+            case 1:
+                clickCss("a[id='a_yes']");
+                break;
+            case 2:
+                clickCss("a[id='a_no']");
+                break;
+            case 3:
+                clickCss("a[id='a_dont_know']");
+                break;
+            case 4:
+                clickCss("a[id='a_probably']");
+                break;
+            case 5:
+                clickCss("a[id='a_probaly_not']");
+                break;
+            case 6:
+                clickCss("a[id='a_cancel_answer']");
+                break;
+
+//default + break заставляє ще раз вибрати, чи все таки потрібно примусово запустити цей switch - case це зробити?
+            //* я так розумію, що в default не треба писати break. Так як break просто перериває перевірку умов.
+// Зробив так, але ще не тестував.
+            default:
+                System.out.println("Этого варианта не существует. Попробуйте еще раз. Спасибо!");
+                chooseYourOption();
+        }
+    }
+
+    public static void confirmYourHero() throws InterruptedException, NoSuchElementException {
+        if (!driver.findElement(By.xpath("//span[@class='proposal-title']")).isEnabled()) {
+            System.out.println("Извините произошла ошибка. Попробуйте еще раз!");
+            return;
+        } else {
+            String answer = driver.findElement(By.xpath("//span[@class='proposal-title']")).getText() + " (" +
+                    driver.findElement(By.xpath("//span[@class='proposal-subtitle']")).getText() + ")";
+
+            System.out.println("Я думаю это: " + answer);
+        }
+
+        confirmProposal();
+    }
+
+    public static void confirmProposal() throws InterruptedException, NoSuchElementException {
+        if (showTwoOptions() == 1) {
+            clickCss("a[id='a_propose_yes']");
+//  Підтягнути текст з Акінатора
+            System.out.println("Я знал! Спасибо за участие!");
+            endGame();
+        } else {
+            clickCss("a[id='a_propose_no']");
+            continueGame();
+        }
+    }
+
+    public static void continueGame() throws InterruptedException, NoSuchElementException{
+        System.out.println(driver.findElement(By.xpath("//span[@class='proposal-title']")).getText());
+        if (showTwoOptions() == 1) {
+            clickCss("a[id='a_continue_yes']");
+            answerQuestions();
+            confirmYourHero();
+        } else {
+            clickCss("a[id='a_continue_no']");
+            enterYourHero();
+        }
+    }
+
+    public static void enterYourHero() throws InterruptedException {
```

Comment:
Could you please refactor this method and separate independent logic to separate methods?